release/sil/sil.bcc-latn.upp_ptwl1/source/sil.bcc-latn.upp_ptwl1.model.kps

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Package>
+  <System>
+    <KeymanDeveloperVersion>13.0.64.0</KeymanDeveloperVersion>
+    <FileVersion>12.0</FileVersion>
+  </System>
+  <Options>
+    <ExecuteProgram></ExecuteProgram>
+    <ReadMeFile>readme.htm</ReadMeFile>
+    <MSIFileName></MSIFileName>
+    <MSIOptions></MSIOptions>
+  </Options>
+  <StartMenu>
+    <Folder></Folder>
+    <Items/>
+  </StartMenu>
+  <Info>
+    <Name URL="">sil.bcc-latn.upp_ptwl1</Name>
+    <Copyright URL="">© 2020 SIL International</Copyright>
+    <Author URL="">RL</Author>
+    <Version URL="">1.0</Version>
+  </Info>
+  <Files>
+    <File>
+      <Name>..\build\sil.bcc-latn.upp_ptwl1.model.js</Name>
+      <Description></Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.js</FileType>
+    </File>
+    <File>
+      <Name>welcome.htm</Name>
+      <Description></Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.htm</FileType>
+    </File>
+    <File>
+      <Name>readme.htm</Name>
+      <Description></Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.htm</FileType>
+    </File>
+  </Files>
+  <Keyboards/>
+  <LexicalModels>
+    <LexicalModel>
+      <Name>sil.bcc-latn.upp_ptwl1</Name>
+      <ID>sil.bcc-latn.upp_ptwl1</ID>
+      <Languages>
```

Comment:
@jahorton I'm not sure KMW uses this yet, but won't it be good to have "RTL" set?

In Keyman Developer --> Lexical Models tab --> there's a checkbox for "Is Right-to-left".

Reply:
It should be used already in 13.0 beta; a recent KMW fix prompted the recent Developer addition of the RTL checkbox.

Reply:
sorry, I mean RTL should be applied in a separate PR for `release/sil/sil.bcc-arab.upp_ptwl1`